packages/apps/webpack.config.js

```diff
@@ -181,7 +181,7 @@ function createWebpack ({ alias = {}, context, name = 'index' }) {
       new HtmlWebpackPlugin({
         inject: true,
         template: path.join(context, `${hasPublic ? 'public/' : ''}${name}.html`),
-        PAGE_TITLE: 'Polkadot/Substrate Portal'
+        PAGE_TITLE: 'Plasmchain Portal'
```

Comment:
What’s about just Plasm or Plasm Network?

Reply:
Plasm Network is fixed.